webapp/src/ts/components/search-bar/search-bar.component.ts

```diff
@@ -24,16 +35,24 @@ export class SearchBarComponent implements AfterContentInit, AfterViewInit, OnDe
   @Input() disabled;
   @Input() showFilter;
   @Input() showSort;
+  @Input() showBarcodeScanner;
   @Input() sortDirection;
   @Input() lastVisitedDateExtras;
   @Output() sort: EventEmitter<any> = new EventEmitter();
   @Output() toggleFilter: EventEmitter<any> = new EventEmitter();
   @Output() search: EventEmitter<any> = new EventEmitter();
 
+  private TELEMETRY_PREFIX = 'search_by_barcode';
```

Comment:
```suggestion
  private readonly TELEMETRY_PREFIX = 'search_by_barcode';
```
tiny change but if this field is supposed to be constant, let's mark it as `readonly`

Reply:
Thanks! I need to get more in the habit of using modifiers.